src/FilenameParsing/LegacyFileIDHelper.php

```diff
@@ -15,6 +17,33 @@
 class LegacyFileIDHelper implements FileIDHelper
 {
     use Injectable;
+    use Configurable;
+
+    /**
+     * List of SilverStripe 3 image method names that can appear in variants. Prior to SilverStripe 3.3, variants were
+     * incoded in the filename with dashes. e.g.: `_resampled/FitW10-sam.jpg` rather than `_resampled/FitW10/sam.jpg`.
+     * @config
+     */
+    private static $ss3_image_variant_methods = [
```

Comment:
By making this configurable, someone who had their own weird variant method can add it to the list.